test/controllers/callback_controller_test.rb

```diff
@@ -25,6 +26,10 @@ class CallbackControllerTest < ActionController::TestCase
     setup do
       @routes = ShopifyApp::Engine.routes
       ShopifyApp.configuration = nil
+      ShopifyApp.configuration.api_key = "api_key"
+      ShopifyApp.configuration.secret = "secret"
+      ShopifyApp.configuration.api_version = "2022-07"
+      ShopifyApp.configuration.scope = "read_orders, write_orders"
```

Comment:
I think the test configurer should load these values with reasonable values, so we wouldn't need to do it here.